### DISCUSSION TITLE
Variant attributes listed multiple times in product detail page

### DISCUSSION THREAD

**Discussion by mbeprint** [saleor/saleor#8000] [Deployment]
Hi everybody,

I am discovering Saleor 3.0. I have somehow deployed the application and now creating the products etc.
But I see a strange behaviour when creating variants. I have attached the image. Basically when I create the variants, attributes are listed multiple times in product detail page.
The product is same day poster and has two variant attributes: print side and print size, and has no product attributes.

Also I can not select the second attribute when selected the first one, or visa versa.

In the demo web site, all products have one variant so I can not see this behaviour. 

I have no idea what I am doing wrong.

I would be grateful if you could help.

Kind regards

![duplicated1](https://user-images.githubusercontent.com/58146076/131689714-37891e61-d1d7-4c44-88df-9eef316561ce.png)

![duplicated2](https://user-images.githubusercontent.com/58146076/131689857-77a35f88-555f-48c6-b881-c3ab26d2b732.png)

![duplicated3](https://user-images.githubusercontent.com/58146076/131689881-0de85fa6-3836-45a6-ae96-dbb891d8a813.png)

![duplicated4](https://user-images.githubusercontent.com/58146076/131689912-9b3315dc-a318-4303-800f-567d8d92d2f7.png)

![duplicated5](https://user-images.githubusercontent.com/58146076/131689942-8e0a18dd-0705-497f-a3b3-0e492862394c.png)

**Comment by mbeprint** [comment]
By the way I am using saleor-3.1.0-a.2, saleor-dashboard-3.1.0-a.2, and saleor-storefront-3.0.0-b.2.

**Reply by patrys** [reply]
@eCzerniak, can you take a look?

**Comment by eCzerniak** [comment]
Hi @mbeprint so I just reproduced it and reported it.

**Reply by mbeprint** [reply]
Thanks for the info. Is there any update for this issue?
